src/NUnitTestAdapter/NUnit.TestAdapter.csproj

```diff
@@ -23,6 +23,11 @@
     <Authors>Charlie Poole, Terje Sandstrom</Authors>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net35'">
+    <!-- Suppress errors from https://github.com/dotnet/sdk/issues/1897-->
+    <DisableHandlePackageFileConflicts>false</DisableHandlePackageFileConflicts>
+  </PropertyGroup>
+
```

Comment:
I believe we're seeing https://github.com/dotnet/sdk/issues/1897 on the CI builds. This change suppresses the extraneous errors from the build logs.